api-report/container-runtime.api.md

```diff
@@ -309,6 +309,12 @@ export interface IClientSummaryWatcher extends IDisposable {
     watchSummary(clientSequenceNumber: number): ISummary;
 }
 
+// @public
+export interface ICompressionRuntimeOptions {
+    readonly compressionEnabled: boolean;
+    readonly minimumSize: number;
```

Comment:
Consider having fewer options - number | undefined is enough. Makes it a bit easier to use feature gates - just one setting to set instead of two. I'd keep interface (to allow future extensions).